index.html

```diff
@@ -3,23 +3,46 @@
   <head>
     <meta charset="utf-8" />
     <title>Rock Paper Scissors</title>
+    <link rel="stylesheet" href="./style.css" />
   </head>
 
   <body>
+    <h1>Rock Paper Scissors Game</h1>
+    <div class="flex-container-1">
+      <button class="btn flex-item rock">Rock</button>
+    </div>
+    <div class="flex-container-2">
+      <button class="btn flex-item paper">Paper</button>
+      <button class="btn flex-item scissors">Scissors</button>
+    </div>
+    <div id="scoreboards">
+      <div class="Score">
+        <h2>Cpu Score</h2>
+        <p></p>
+      </div>
+      <div class="Score">
+        <h2>Player Score</h2>
+        <p></p>
+      </div>
+    </div>
+
     <script>
-      /* create variables to store cpu and player wins */
       let cpuWins = 0;
       let playerWins = 0;
-
-      /*create variable to store cpu choice as rock, paper or scissors */
       let cpuChoice;
 
-      /*create variable to choose a random number then store it in a variable */
-      /*change value of cpu choice variable depending on value of random number variable */
-      /*create function to run this code and make a different cpu choice each time function is run */
+      const cpuScore = document.querySelectorAll("p");
+      let updateCpuScore = function () {
+        cpuScore[0].textContent = cpuWins;
+      };
+
+      const playerScore = document.querySelectorAll("p");
+      let updatePlayerScore = function () {
+        playerScore[1].textContent = playerWins;
+      };
+
```

Comment:
this is very fragile. If someone adds another p element at the top then this will break. What can you use here instead?

Reply:
I'll add different ID's to them and link the variables to them with getElementbyId